### ISSUE TITLE
Provide standalone language server

### ISSUE THREAD

**Issue by shyim** [DEVSENSE/phptools-docs#543]
I want to use the PHP language server with another editor like https://zed.dev/ . It would be nice if there were some kind of place where the language server can be downloaded standalone.

**Comment by dhenfie** [comment]
@jakubmisek it seems like zed also uses the same LSP as VSCODE.

**Comment by jakubmisek** [comment]
> @jakubmisek it seems like zed also uses the same LSP as VSCODE.

Yes, the LSP itself (https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/) is the same. But we have additional functionality that is implemented outside LSP, since it was not enough.

**Comment by ADmad** [comment]
Would be really nice to be able to use this with zed editor.

**Comment by Spriz** [comment]
Or directly in vim/nvim! 💌

**Comment by jakubmisek** [comment]
We have prepared a standalone LS as a Node.js package.

The stable version will be released in June, and we have prepared pull requests for other IDEs, including Zed and Neovim. Vim and IdeaJ later.

**Comment by jakubmisek** [comment]
update:

- standalone LS: https://www.npmjs.com/package/devsense-php-ls
- Zed extension in progress: https://github.com/zed-extensions/php/pull/34

**Comment by haringsrob** [comment]
This is absolutely brilliant! It was the only thing keeping me on vscode/cursor. Massive thank you!

**Comment by jakubmisek** [comment]
@haringsrob yes! thank you. We have opened a pull request on lspconfig repo: https://github.com/neovim/nvim-lspconfig/pull/3966
